ZFSin/zfs/lib/libzfs/libzfs_import.c

```diff
@@ -1535,17 +1552,20 @@ zpool_open_func_win(void *arg)
 			FILE_ATTRIBUTE_NORMAL /*| FILE_FLAG_OVERLAPPED*/,
 			NULL);
 		if (fd == INVALID_HANDLE_VALUE) {
-			int error = GetLastError();
+			DWORD error = GetLastError();
+			TraceWrite("func return.CreateFile() failed with error %lu.FullPath:%s ThreadID:%lu [%s:%d]", error, fullpath, threadID, __func__, __LINE__);
 			return;
 		}
 
+		TraceWrite("createFile() success. Get GetFileDriveSize().FullPath:%s ThreadID:%lu [%s:%d]", fullpath, threadID, __func__, __LINE__);
```

Comment:
If there is a time gap between line 1560 and line 1567, then we won't be able to find which of the three functions (GetFileDriveSize, GetFileType, GetDriveType) took time right? So should we add few prints between these functions?